release-notes/dashboards-observability.release-notes-2.11.1.0.md

```diff
@@ -0,0 +1,41 @@
+## Version 2.11.1 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.11.0
+
+### Features
+
+- [Feature] Match discover look and feel by @opensearch-trigger-bot in ([#1179](https://github.com/opensearch-project/dashboards-observability/pull/1179)
```

Comment:
Nit: would be nice to be able to link the original author instead of the backport bot